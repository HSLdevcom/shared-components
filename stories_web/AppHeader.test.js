import initStoryshots from '@storybook/addon-storyshots';

initStoryshots({
  storyKindRegex: /^AppHeader$/
});
