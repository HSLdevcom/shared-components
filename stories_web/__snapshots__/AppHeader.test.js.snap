// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots AppHeader default 1`] = `
<div
  className="sc-bwzfXH gXRUvk"
>
  <p>
    Some content
  </p>
</div>
`;
